### DISCUSSION TITLE
Torus widget still showing after configuring `showTorusButton: false`

### DISCUSSION THREAD

**Discussion by pavelescurazvan** [Web3Auth/Web3Auth#974] [Help]
Hello,

After setting `showTorusButton: false` the torus widget is still showing as a floating button in the bottom left part of the screen.

Any idea what might be going on?

Screenshot with configuration:
<img width="946" alt="Screenshot 2022-12-01 at 13 05 30" src="https://user-images.githubusercontent.com/25104492/205037404-fd3bc4f5-9328-44e5-be4e-eff76bcd019e.png">

Thank you,
Razvan

**Comment by shahbaz17** [comment]
Hey @pavelescurazvan 

Please update to the latest package. And let us know if this issue still persists.

**Reply by lomads** [reply]
Using 5.2.0. Still same issue
